Src/AutoNSubstitute.sln

```diff
@@ -11,6 +11,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AutoNSubstituteUnitTest", "
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AutoNSubstitute", "AutoNSubstitute\AutoNSubstitute.csproj", "{189C9474-B406-4356-A367-FBC882E9BA4B}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AutoFixture.xUnit.net", "AutoFixture.xUnit.net\AutoFixture.xUnit.net.csproj", "{266B9F0E-28E5-47CA-9816-ACB6737E7E4D}"
```

Comment:
I've added scenario tests to illustrate the usage of the new API because this approach was [already common in AutoFixture](https://github.com/AutoFixture/AutoFixture/search?utf8=%E2%9C%93&q=scenario). I don't have a strong preference on this matter. It's your call. 

To answer your questions, I don't find scenario tests to be particularly useful after the design has settled down and would be happy to remove them before the PR is merged. I like the idea of moving scenario tests to a separate solution less. It would reduce their value by separating them from the APIs they are meant to help you learn. Moving scenario tests to a separate project within the NSubstitute solution may be a better option.